datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py

```diff
@@ -42,12 +53,17 @@ def __init__(self, *args, **kwargs):
         self.default_namespace = default_namespace
 
         # pre-generate the scraper configurations
+
         if 'instances' in kwargs:
             instances = kwargs['instances']
         elif len(args) == 4:
+            # instances from legacy signature
             instances = args[3]
-        else:
+        elif isinstance(args[2], (tuple, list)):
+            # instances from new signature
```

Comment:
```suggestion
            # instances from agent 6 signature
```
Lets avoid using old and new because new will be old one day